mobilecoind-json/src/data_types.rs

```diff
@@ -841,7 +841,10 @@ impl From<&RingMLSAG> for JsonRingMLSAG {
 pub struct JsonSignatureRctBulletproofs {
     pub ring_signatures: Vec<JsonRingMLSAG>,
     pub pseudo_output_commitments: Vec<String>,
-    pub range_proofs: String,
+    pub range_proof_bytes: String,
+    pub range_proofs: Vec<String>,
+    pub pseudo_output_token_ids: Vec<u64>,
```

Comment:
I think `u64` is a problem in some JSON parsers, which is why in a bunch of places it is encoded as a string ;(

Reply:
i see -- thank you